src/commands/codepush/deployment/list.ts

```diff
@@ -80,6 +90,10 @@ export default class CodePushDeploymentListListCommand extends AppCommand {
       deployments,
       async (deployment) => {
         if (formatIsJson()) {
+          if (!deployment.latestRelease || this.skipFetchingDeploymentMetrics) {
+            return deployment;
+          }
+
           const metricsJSON: models.CodePushReleaseMetric = await this.generateMetricsJSON(deployment, client);
 
           if (metricsJSON && deployment.latestRelease) {
```

Comment:
Since you added the `!deployment.latestRelease` condition above, let's remove it from this condition:

```js
if (metricsJSON && deployment.latestRelease)
```

->

```js
if (metricsJSON)
```

Reply:
removed the unwanted check